kolibri/plugins/perseus_viewer/assets/src/views/PerseusRendererIndex.vue

```diff
@@ -144,7 +144,7 @@
         return this.windowBreakpoint < 3;
       },
       usesTouch() {
-        return isTouchDevice;
+        return isTouchDevice && localStorage.getItem('mouseUsed') !== 'true';
```

Comment:
As I commented above, let's add a browserInfo exportable variable that will define this value - so that the implementation details are hidden from the code consumer here.